seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java

```diff
@@ -250,6 +250,9 @@ private float convertToFloat(JsonNode jsonNode) {
     }
 
     private LocalDate convertToLocalDate(JsonNode jsonNode, String fieldName) {
+        if ("".equals(jsonNode.asText())) {
+            return null;
+        }
```

Comment:
I want to confirm one thing first, is the data in kafka `""` or `null`? If it is `null`, we should continue. If it is `""`, I think it is normal to report an error. Please refer https://github.com/apache/seatunnel/pull/7181

Reply:
> I want to confirm one thing first, is the data in kafka `""` or `null`? If it is `null`, we should continue. If it is `""`, I think it is normal to report an error. Please refer #7181

I have read your pr, your design is very good, it is no problem to throw wrong theory, but there is another scenario you need to consider, that is, a batch of data sent from upstream, for example, {"date_field":"2023-02-01"},{"date_field":""}, should we return empty strings or pass null to the downstream database instead of parsing them?

Reply:
> I want to confirm one thing first, is the data in kafka `""` or `null`? If it is `null`, we should continue. If it is `""`, I think it is normal to report an error. Please refer #7181
I task this batch of tasks can not be due to an empty string caused by the task error directly quit

Reply:
I think we can add an option to control this behavior. If a field cannot be parsed, return `null` or throw an error. Just like hive. Not just datetime type, this rule is suitable for any type of parsing. cc @hailin0

Reply:
> I think we can add an option to control this behavior. If a field cannot be parsed, return `null` or throw an error. Just like hive. Not just datetime type, this rule is suitable for any type of parsing. cc @hailin0

Good idea. You can control whether to throw an exception or return null with parameters